bin/utils/maven/java-petstore-server-all-build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
Build projects generated by /bin/java-petstore-server-all.sh
Does not include swt-based java-play-framework* projects.